src/services/objection.service.ts

```diff
@@ -103,3 +103,21 @@ const getValuesForApiCall = (session: Session) => {
   const token: string = retrieveAccessTokenFromSession(session);
   return { objectionId, companyNumber, token };
 };
+
+/**
+ * Gets objection details for the company and objection present in the web session.
+ *
+ * @param {Session} session the web session
+ *
+ * @returns {Objection} details of the existing objection
+ */
+export const getObjection = async (session: Session): Promise<Objection> => {
+  const companyProfileInSession: ObjectionCompanyProfile = retrieveCompanyProfileFromObjectionSession(session);
```

Comment:
you should be able to replace these 4 lines with 
const { objectionId, companyNumber, token } = getValuesForApiCall(session);
after the master merge.
Also move the function up above the non exported one to keep it grouped with the others